civictechprojects/static/css/partials/_LogInController.scss

```
@@ -88,3 +88,20 @@
   height: 55px;
   cursor: pointer;
 }
+
+.LogInController-google svg{
+  width: 40px;
+  height: 50px;
+}
```

```suggestion
.LogInController-google svg {
  padding: 5px 0 4px 0;
}
```
Changing the svg size itself changes the entire size of the containing box (since the 55px SVG height effectively sets the height, as the tallest element) so instead let's pad the SVG to 'push' the image into the shape we want while retaining the 55px height. This prevents the Google box from being 5px shorter than Github, LinkedIn, etc.